mlrun/config.py

```diff
@@ -110,6 +110,12 @@
             "mlrun_version_specifier": "",
         },
     },
+    "vault": {
```

Comment:
Create under the httpdb block a new block `secrets_provider` and under it add this `vault` block

Reply:
Don't see why it should be under httpdb. I think should name the block `secret_providers` and put it there (though it will generate some really ugly env variables - `MLRUN_SECRET_PROVIDERS__VAULT__TOKEN_PATH`...